frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json

```diff
@@ -458,7 +458,7 @@
   "Could not create Persistent volume claim": "Could not create Persistent volume claim",
   "No base image specified": "No base image specified",
   "You cannot add source to this template because it is missing base image specification.": "You cannot add source to this template because it is missing base image specification.",
-  "This data can be found in <2>Storage > Persistent volume claims > {baseImageName}</2> under the <5>{baseImageNamespace}</5> project.": "This data can be found in <2>Storage > Persistent volume claims > {baseImageName}</2> under the <5>{baseImageNamespace}</5> project.",
+  "This data can be found in <2>Storage &gt; Persistent volume claims &gt; {baseImageName}</2> under the <5>{baseImageNamespace}</5> project.": "This data can be found in <2>Storage &gt; Persistent volume claims &gt; {baseImageName}</2> under the <5>{baseImageNamespace}</5> project.",
```

Comment:
The `kubevirt` code is orphaned, so I'd omit this change if the build will allow it.

Reply:
will update the i18n script not to take care about the pkg